Project8/src/TransportSystem/Transport.java

```diff
@@ -0,0 +1,5 @@
+package TransportSystem;
+
+public interface Transport {
+    void move();
```

Comment:
make move not only print current distance, but to move distance

Reply:
Fixed